VAMobile/src/screens/SplashScreen/SplashScreen.tsx

```diff
@@ -11,19 +11,35 @@ function SplashScreen({}: SplashScreenProps) {
   const splashStyles: ViewStyle = {
     flexGrow: 1,
     justifyContent: 'center',
-    backgroundColor: theme.colors.background.splashScreen,
+    backgroundColor: theme.colors.background.loginScreen,
+  }
+  const logoProps: ImageProps = {
+    width: 254,
+    height: 57,
   }
   const isPortrait = useOrientation()
 
   return (
     <VAScrollView {...testIdProps('Splash-page')} contentContainerStyle={splashStyles} removeInsets={true}>
+      <StatusBar
+        translucent
+        barStyle={theme.mode === 'dark' ? 'light-content' : 'dark-content'}
+        backgroundColor={theme.colors.background.main}
+      />
```

Comment:
Stole this code from the screen templates to ensure the header is changing with the theme.